index.html

```diff
@@ -1,3 +1,7 @@
+
+
+
+
 <html>
     <head>
```

Comment:
- [ ]  It seems your html file is not set up properly.
Kindly  the meta tags to the <head>  tage visit this [site](https://www.w3schools.com/tags/tag_meta.asp) for more info
This is to avoid  Web related errors to your code

>

Reply:
## OPTIONAL

- [ ] Nice Job @amklass11 👍. Always make sure that the comments are clear and easy to understand. 
Example: Your Html file is not set up properly. Please, I suggest that you add ... to avoid Web-related errors. Kindly visit this site for more information.